internal/addon/addon.go

```diff
@@ -31,6 +36,25 @@ func AgentHealthProber() *agent.HealthProber {
 		Type: agent.HealthProberTypeWork,
 		WorkProber: &agent.WorkHealthProber{
 			ProbeFields: []agent.ProbeField{
+				{
+					ResourceIdentifier: workv1.ResourceIdentifier{
+						Group:     prometheusalpha1.SchemeGroupVersion.Group,
+						Resource:  PrometheusAgentResource,
+						Name:      PPAName,
+						Namespace: InstallNamespace,
+					},
+					ProbeRules: []workv1.FeedbackRule{
+						{
+							Type: workv1.JSONPathsType,
+							JsonPaths: []workv1.JsonPath{
+								{
+									Name: paProbeKey,
+									Path: paProbePath,
+								},
+							},
+						},
+					},
+				},
```

Comment:
@thibaultmg I've added a probe to the Platform prometheus, feel free to request any changes in this. It I can also add for UWM if you want but I would also have to update the HealthChecker